### PR TITLE
Fix grammar

### PR DESCRIPTION
**Pull request by ic-768**
Fixed minor grammar issues in transforms section of documentation

### CODE REVIEW

**Review by alanbsmith — APPROVED** [review]
👍🏼

**Review by dbanksdesign — APPROVED** [review]

File: docs/transforms.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ There are 3 types of transforms: attribute, name, and value.
 
 **Attribute:** An attribute transform adds to the attributes object on a design token. This is for including any meta-data about a design token such as it's CTI or other information.
 
-**Name:** A name transform transform the name of a design token. You should really only be apply one name transformer because they will override each other if you use more than one.
+**Name:** A name transform transforms the name of a design token. You should really only be applying one name transformer because they will override each other if you use more than one.
```
Comment (dbanksdesign) [inline docs/transforms.md:36]: Can you update this file: https://github.com/amzn/style-dictionary/blob/main/scripts/handlebars/templates/transforms.hbs  the docs/transforms.md file is generated via a script based on that handlebars file

**Review by dbanksdesign — CHANGES_REQUESTED** [review]
Need to update another file so this change doesn't get lost when we release

File: docs/transforms.md (hunk 1 above)
Comment (ic-768) [reply docs/transforms.md:36]: Done!

**Comment by jorenbroekema** [comment]
@dbanksdesign  can you change your review to approve?

I also went ahead and forward ported this to v4 already :)

**Review by dbanksdesign — APPROVED** [review]

**Pull request merged**